README.md

```diff
@@ -103,6 +103,14 @@ bundle exec rspec spec/PATH/TO/FILE_spec.rb
 bundle exec rspec spec/PATH/TO/DIR
 ```
 
+## Camunda-ext
```

Comment:
Let's create `Links` section and with smth like 
`[Repo with helper classes for BPMN development](https://github.com/latera/camunda-ext)`